app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self,
+                 name: str,
+                 age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    ls_of_friends = [Person(person["name"], person["age"])for person in people]
+    for key, value in Person.people.items():
+        for person in people:
+            if key == person["name"]\
+                    and "wife" in person.keys()\
+                    and person["wife"]:
+                value.wife = Person.people[person["wife"]]
+            if key == person["name"]\
+                    and "husband" in person.keys()\
+                    and person["husband"]:
+                value.husband = Person.people[person["husband"]]
```

Comment:
U can get wife or husband from person using person.get("wife") or person.get("husband")
So, you needn't use first loop (for key, value in Person.people.items()) and large comparison